notebooks/lesson07_classification.ipynb

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
<p>is there a comment missing before grid search?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>

Reply:
<p>good catch - fixed!</p>